### DISCUSSION TITLE
problem in installing new version of nuclei in kali or parrot os

### DISCUSSION THREAD

**Discussion by Adarshhacker** [projectdiscovery/nuclei#3754] [General]
┌──(root㉿hacker)-[/home/…/nuclei/v2/cmd/nuclei]
└─# go build
# github.com/projectdiscovery/interactsh/pkg/client
/root/go/pkg/mod/github.com/projectdiscovery/interactsh@v1.1.4/pkg/client/client.go:327:17: undefined: errors.Join
note: module requires Go 1.20

this is the issue which i am facing i tried many ways and it is suggesting go 1.20 has not been released yet
![Screenshot 2023-05-30 055945](https://github.com/projectdiscovery/nuclei/assets/100475906/a9d00114-e1ad-4669-99fa-810a1879aa17)

**Comment by ehsandeep** [comment]
@Adarshhacker as the note says: `note: module requires Go 1.20` you need to have go 1.20.* installed on your system to install using go, or you can use [pdtm](https://github.com/projectdiscovery/pdtm) to download the complied binary.

You can check you current version with `go version`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
